h5p-row.css

```
@@ -1,3 +1,12 @@
-.h5p-row {
-  background: salmon;
+.h5p-row-wrapper {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
 }
+
+.h5p-column {
```

Change to match the suggestion below.
```suggestion
.h5p-row-content {
```